### DISCUSSION TITLE
Simplify Governance

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#79] [Ideas] [answered]
## Problem

The current governance was adopted from the Flux GOVERNANCE doc, however the needs for this WG are most likely much simpler. This was already informally approved in general in working group meetings.

For history, see:
- https://github.com/gitops-working-group/gitops-working-group-old/pull/40
- https://github.com/gitops-working-group/gitops-working-group/pull/1

## Proposed Solution

I propose we dramatically simplify the governance doc for this WG. Note the requirements for this may differ depending on the outcome of #32, but in general according to the existing governance doc we will just need approval by the maintainers for a simpler version.

## Status

- [x] Raise this discussion in CNCF `#wg-gitops` Slack
- [x] Raise discussion at working group meetings
- [x] @scottrigby Create a PR with proposed simplifications for discussion / approval / merging into main - 👀  in review https://github.com/gitops-working-group/gitops-working-group/pull/125
- [x] Ensure any changes are announced at the next working group meeting

**Comment by bricef** [comment]
Agreed on simplifying governance 👍 Will you put together the PR @scottrigby? We can work through specifics there.

**Reply by scottrigby** [reply]
~Yes! This week ✨~

**Reply by scottrigby** [reply]
Update: There are now several PRs addressing different parts of this:
- open-gitops/gitops-working-group#64
- #84
- #86

This week planning to collaborate on another, broader follow-up simplification PR

**Reply by scottrigby** [reply]
See https://github.com/gitops-working-group/gitops-working-group/pull/125

**Comment by tonit** [comment]
What's the status here? How might one help here?

**Reply by scottrigby** [reply]
Focusing on this WG and sandbox project this week. Did a bit over the weekend but trying to make sure I enjoy the first real glimmers of spring here in Brooklyn. I'll be around Slack Monday if you want to connect. There are a few steps here

**Comment by scottrigby** [comment, marked as answer]
This has been complete for some time. See links above, and from #125:
- https://github.com/gitops-working-group/gitops-working-group/blob/main/GOVERNANCE.md
- https://github.com/gitops-working-group/gitops-working-group/blob/main/charter.md
